index.js

```diff
@@ -1,13 +1,12 @@
 // Collect client and environment information
+const utils = require('@percy/sdk-utils');
```

Comment:
Should move this to the top with an extra space beneath it since it isn't relevant to collecting client and environment information

```suggestion
const utils = require('@percy/sdk-utils');

// Collect client and environment information
```